src/Webform/ThankYouMessage/styled/message.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 export default styled.h2`
-  background-color: ${p => p.theme.primaryColor};
+  background-color: transparent;
```

Comment:
Just to check.. Isn't this the default? If so, we should remove this line of re-setting it's default value.

```suggestion
```

Reply:
Yeah, this can probably be removed!